scrnaseq.Rmd

```diff
@@ -98,15 +178,15 @@ param$path_data = data.frame(name=c("pbmc_10x","pbmc_smartseq2"),
 
 # Downsample data to at most n cells per sample (mainly for tests)
 #   NULL to deactivate
-param$downsample_cells_n = 100
+param$downsample_cells_n = NULL
```

Comment:
We can keep it at 100 until the release.

Reply:
Done